site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,233 @@
+                </div>
+                <div id="inputCompanyFeedback" class="invalid-feedback">
+                  Please enter a valid company name.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="selectBusiness" id="selectBusinessLabel" class="form-label is-required">Business
+                  type</label>
+                <select class="form-select" id="selectBusiness"
+                  aria-labelledby="selectBusinessLabel selectBusinessFeedback" required>
+                  <option selected disabled value="" aria-hidden="true">Select a type</option>
+                  <option value="1">Sole proprietorship</option>
+                  <option value="2">Partnership</option>
+                  <option value="3">Corporation</option>
+                  <option value="4">Limited liability company</option>
+                </select>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="selectBusinessFeedback" class="invalid-feedback">
+                  Please select a valid business type.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="inputWebsite" id="inputWebsiteLabel" class="form-label">Website</label>
+                <input type="text" class="form-control" id="inputWebsite"
+                  aria-labelledby="inputWebsiteLabel inputWebsiteFeedback">
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="inputWebsiteFeedback" class="invalid-feedback">
+                  Please enter a valid website.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="textareaMessage" id="textareaMessageLabel" class="form-label is-required">Message</label>
+                <button type="button" class="form-helper" data-bs-toggle="tooltip" data-bs-placement="top" data-bs-title="Help for text area">
+                  <span class="visually-hidden">Helper for text area</span>
+                </button>
+                <textarea class="form-control" id="textareaMessage" rows="5"
+                  placeholder="Describe your business and your motivation in becoming an Orange reseller"
+                  aria-labelledby="textareaMessageLabel textareaMessageFeedback" required></textarea>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="textareaMessageFeedback" class="invalid-feedback">
+                  Please enter a valid message.
+                </div>
+              </div>
+              <button type="submit" class="btn btn-primary mt-2">Submit</button>
+            </div>
+          </div>
+        </form>
+      </div>
+    </div>
+  </div>
+</main>
+<footer class="footer bg-dark navbar-dark">
+  <h2 class="visually-hidden">Sitemap &amp; information</h2>
```

Comment:
I know that "&" is forbidden when it's written at Orange and must be replaced by "and". IDK if it is the same thing for accessibility but I suppose that there's more chance that screen readers say it well if we make the following modification here:
```suggestion
  <h2 class="visually-hidden">Sitemap and information</h2>
```